samples/extensions/raytracing_extended/raytracing_extended.h

```diff
@@ -21,7 +21,7 @@
 
 #pragma once
 
-#define USE_FRAMEWORK_ACCELERATION_STRUCTURE
+//#define USE_FRAMEWORK_ACCELERATION_STRUCTURE
```

Comment:
why make it optional?

Reply:
It's optional because it's the first sample to use the framework for acceleration structure.  I felt it's good in the first sample to show an equivalent how to build with and without the framework.  This can be both instructive and a nice validation of expectations.